web/src/main/java/org/openmrs/web/filter/initialization/InitializationFilter.java

```diff
@@ -1707,6 +1707,10 @@ public void executing(ChangeSet changeSet, int numChangeSetsToRun) {
 						// start openmrs
 						try {
 							UpdateFilter.setUpdatesRequired(false);
+							
+							// rebuild lucene search indexes
+							Context.updateSearchIndex();
```

Comment:
I guess this is fine, the  GP gets cleared after each run so there will be no second run happening immediately after